querystring/parse.js

```diff
@@ -8,7 +8,12 @@ module.exports = function(string) {
 	for (var i = 0; i < entries.length; i++) {
 		var entry = entries[i].split("=")
 		var key = decodeURIComponent(entry[0])
-		var value = entry.length === 2 ? decodeURIComponent(entry[1]) : ""
+		var value = "";
+		try{
+			value = entry.length === 2 ? decodeURIComponent(entry[1]) : "" 
```

Comment:
maybe creating a `decodeURIComponentSave` would be easier
```javascript
function decodeURIComponentLoose(str) {
	try {
			return decodeURIComponent(str)
	} catch(err) {
			return str
	}
}
```

Anyone sees any security issues here?